.github/workflows/bleeding-edge.yml

```diff
@@ -0,0 +1,71 @@
+# This workflow installs dependencies from master
+
+name: ETS from source
+
+# TODO: Switch to schedule!
+on: push
+
+jobs:
+  test-ets:
+    strategy:
+      matrix:
+        # Note that Ubuntu 20 cannot be used because its SWIG version is 4.0
+        # (enthought/enable#360)
+        os: [ubuntu-18.04]
+        toolkit: ['pyside2', 'wx']
+        python-version: [3.8]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Check out
+        uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install PySide2 (from PyPI) dependencies for Linux
+        run: |
+          sudo apt-get update
+          sudo apt-get install qt5-default
+          sudo apt-get install libxkbcommon-x11-0
+          sudo apt-get install libxcb-icccm4
+          sudo apt-get install libxcb-image0
+          sudo apt-get install libxcb-keysyms1
+          sudo apt-get install libxcb-randr0
+          sudo apt-get install libxcb-render-util0
+          sudo apt-get install libxcb-xinerama0
```

Comment:
Are all of these needed?

Reply:
Yes, because this workflow uses PySide2 from PyPI. See this build where I have only included the linux packages from the other workflow where PySide2 from EDM is used: https://github.com/enthought/enable/runs/1697094819